app/Http/Middleware/Authenticate.php

```diff
@@ -12,6 +12,6 @@ class Authenticate extends Middleware
      */
     protected function redirectTo(Request $request): ?string
     {
-        return $request->expectsJson() ? null : route('login');
+        return $request->expectsJson() ? null : route('please_login');
```

Comment:
i like the addition of "please" =)) Just make sure the name of the route is correct :D